lib/models/user.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:json_annotation/json_annotation.dart';
```

Comment:
This file should be placed in `api/user/models/user.dart`
And don't forget to add barrel file `models.dart`